index.html

```diff
@@ -2947,10 +2945,10 @@ <h3>
 
           <p>
   If <code>returnFullDocuments</code> was set to true, a successful query will result in a standard HTTP 200 response with a list
-  of <a>EncryptedDocuments</a> that contain the value:
+  of <a>EncryptedDocuments</a> that contain the attribute key:
```

Comment:
attribute key or attribute name or attribute value? :)

Reply:
I changed it to say "...that match the query" to be more consistent with the previous examples - so you just have to check the example to see what type of query it is (attribute key in this case)